### DISCUSSION TITLE
One Track Mode

### DISCUSSION THREAD

**Discussion by ostseeturtle** [advplyr/audiobookshelf#1941] [Ideas]
Hi,

audiobookshelf is great for audiobooks, but ... I have a large collection of audiobook series - with one episode as one track. For this audiobookshelf is not as good as it can be. For this it would be great to have something like a file mode with

* cover per track
* autobookmark per track

The only app at the moment who makes it perfect is nPlayer plus, an audio and video player. For the moment I would find it great to have for audiobooks a podcast view with covers. This would be great. :)

**Comment by advplyr** [comment]
Why not separate them?
